content/altinn-studio/guides/development/signing/api-signing/_index.en.md

```diff
@@ -0,0 +1,110 @@
+---
+title: API based signing
+linktitle: API based signing
+description: Follow these steps to implement API based signing in your app.
+tags: [signering]
+weight: 53
+aliases:
+- /nb/altinn-studio/guides/signing/api-signing
+---
+
+## What do we mean by API based signing?
+
+{{% insert "content/altinn-studio/guides/development/signing/api-signing/intro.en.md" %}}
+
+## Prerequisites
+
+If the service should send a signature receipt to the inbox of the signee, the Altinn message service (Correspondence) must be enabled.
+
+## Configuring signing
+
+You can see how to configure signing in an app using the guides found [here](/altinn-studio/guides/development/signing/).
+API based signing has the same requirements for configuration, except the layout-set for the signing step can be simpler.
+
+## The most important API calls
+
+### Perform single signing
+If only one person should sign in a signing step, you can use the "process next" endpoint:
+
+`PUT /{org}/{app}/instances/{instanceOwnerPartyId}/{instanceGuid}/process/next`
+  
+Body:
+```json
+{
+  "action": "sign"
+}
+```
+
+In this case, the process will automatically navigate to the next step once the signature has been saved.
+If more people should sign in sequence after each other, more signature steps can be added in a chain.
+
+### Perform parallel signing
+
+If it should be possible for multiple signees to sign in parallel, the following endpoint can be used instead of "process next":
+
+`POST /{org}/{app}/instances/{instanceOwnerPartyId}/{instanceGuid}/actions`
+
+Body:
+```json
+{
+  "action": "sign"
+}
+```
+
+In this case, the process will still be in the signing step after the signature has been saved, and more signees can sign before the process is continued using a "process next" call.
+
+### Information about signees and signatures
+
+If [runtime delegated signing](/altinn-studio/guides/development/signing/runtime-delegated-signing/) is configured, the following endpoint can be used to get a list of the signees and their signing status.
+The endpoint can also be used when runtime delegated signing is not used, but then it will only return signees that have already performed the sign action. It will then function as a signature list.
```

Comment:
```suggestion
The endpoint can also be used when runtime-delegated signing is not used, but then it will only return signees that have already performed the sign action. It will then function as a signature list.
```